torch/csrc/jit/codegen/cuda/lower_loops.cpp

```diff
@@ -536,6 +561,27 @@ void LoopNestGenerator::generate(const std::vector<Expr*>& exprs) {
   }
 }
 
+void LoopNestGenerator::cleanSharedMemory() {
+  for (auto& item : smem_) {
+    item.second = false;
+  }
+}
+
+void LoopNestGenerator::modifySharedMemory(Val* const key) {
```

Comment:
please remove `const` (search for the slack thread on why const parameters are strongly discouraged)

Reply:
done